sdk/keyvault/Azure.Security.KeyVault.Keys/src/JsonWebKey.cs

```diff
@@ -20,7 +18,7 @@ public class JsonWebKey : Model
         public string KeyId { get; set; }
 
         /// <summary>
-        /// Gets or sets supported JsonWebKey key types (kty) for Elliptic
+        /// Supported JsonWebKey key types (kty) for Elliptic
```

Comment:
I'm not sure that I understand the context, based on the comment.   Specifically, is there any guidance that we could point to on the trade-offs for the choices or why I would want one over the other?

Reply:
Added more information